README.md

```diff
@@ -452,42 +507,43 @@ You will need the following things properly installed on your computer.
 
 ## Installation
 
-* `git clone <repository-url>` this repository
+* `git clone https://github.com/lblod/frontend-embeddable-notule-editor.git` this repository
 * `cd frontend-embeddable-notule-editor`
 * `npm install`
 
-## Running / Development
-
+## Running
 * `ember serve`
-* Visit your app at [http://localhost:4200/test.html](http://localhost:4200/test.html).
-* Visit your tests at [http://localhost:4200/tests](http://localhost:4200/tests).
-
-### Code Generators
-
-Make use of the many generators for code, try `ember help generate` for more details
-
-### Running Tests
-
-* `ember test`
-* `ember test --server`
-
+* Visit an example of the embeddable at [http://localhost:4200/test.html](http://localhost:4200/test.html).
 ### Linting
-
 * `npm run lint:hbs`
 * `npm run lint:js`
 * `npm run lint:js -- --fix`
 
 ### Building
-
 * `ember build` (development)
 * `ember build --environment production` (production)
+### Releasing
+Take special care when releasing a new version of this.
+- The changelog is automatically created using the PRs. 
+- To avoid a connection to Github, build files are not automatically added to the release
+	- Add the files under `dist/assets` from a production build to the release in a .zip archive. 
```

Comment:
I think it should be all files under `dist`?

Reply:
I'm actually not sure. Are the other files referenced in the assets files, and that's the reason they are needed?
It is not really specified in the readme (now or before) if you should also "host" those in the same place, or if only the ones under `assets` are needed and the rest can be ignored.

Either way (if they are needed or not needed), it should be more explicit, so definitely needs an update around that.

Reply:
In the previous releases, we always bundled the whole `dist` folder I think. But I don't know if the files outside of `assets` are needed.

Reply:
I did a small test and as far as I can tell they are not needed, everything seems to works as expected. I might be missing something though :shrug:

Reply:
ok :) makes sense!

Reply:
added a bit more explicit explanation, so there is more consistency between the different ways of using Embeddable.